docs/meetings/team/2022_11_29.md

```diff
@@ -0,0 +1,58 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 9:30am
+
+**End Time** 10:40am
+
+**Location** Discord
+
+**Date** Sept 16th 2022
+
+## 👋 Attendence
+
+1. Matt
+2. Veronica
+3. Logan
+4. Edouard
+
+***Attendence Comments***
+
+## 🧸 Check-In
+
+- Dad joke of the day
+
+## Announcements
+
+- GitStats are due Friday
+- Peer testing report is due (next) Friday
+  - Prepare for the report this week for peer testing
+- Peer Testing on Wed and Friday
+
+## Last Week (Restrospective)
+
+- Feedback from usability tasks
+  - TA really liked our tasks
+  - We should have more working cases
+    - Should make questions for user testing
+- Unit tests
+  - 3 pass, expect corrupt wav file case fails
+
+## This Week  
+
+### Peer Testing
+
+- Peer testing is complete, made a release for peer testing
+- Have the peers use the Kukerella's front-end:
+  - If there is more time then the user should create audio files from the site, they would re-name the file
+- Including questions about the video and voice-synch
+- All cases have been tested!
+
+## Client Meeting
+
+- Not sure...
+
+## Additional Notes
+
+- API might not be the best option, may cause some issues for our package and the way it runs
```

Comment:
Also, it would likely be just as easy for Kukarella to implement our system as a package as it would be for them to implement as an api, and additionally Im thinking there may be additional processing time required to send audio & text files via http and then return an mp4 file via http... This is something we should bring up at the next client meeting. Also hosting it as a separate API would likely involve extra hosting costs? 

I'm just not sure if there are any advantages to building out our system as an API... Especially since we have intentionally built everything on the same tech stack which they use

Reply:
This is a really good point for us to bring up this Friday!

I think we should also determine if this Friday is our last client meeting until January